Audit scanner cronjob has invalid value

https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#schedule-syntax
Valid value is `minute (0 - 59)`

https://github.com/kubewarden/helm-charts/blob/main/charts/kubewarden-controller/values.yaml#L339
Our chart value is `*/60 * * * *`, which is invalid

```
kubectl get cronjobs --all-namespaces \
                           -o custom-columns="NAMESPACE:.metadata.namespace,NAME:.metadata.name,SCHEDULE:.spec.schedule"
NAMESPACE    NAME            SCHEDULE
kubewarden   audit-scanner   */60 * * * *
```

```
k logs -n prometheus prometheus-kube-state-metrics-55d8bdb7dc-gk475
I0522 06:42:43.558677       1 wrapper.go:151] "Starting kube-state-metrics"
W0522 06:42:43.558913       1 client_config.go:682] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.
I0522 06:42:43.559083       1 server.go:223] "Used resources" resources=["replicationcontrollers","deployments","ingresses","jobs","pods","resourcequotas","statefulsets","storageclasses","volumeattachments","configmaps","endpointslices","mutatingwebhookconfigurations","replicasets","secrets","daemonsets","horizontalpodautoscalers","leases","networkpolicies","persistentvolumes","services","validatingwebhookconfigurations","certificatesigningrequests","cronjobs","limitranges","namespaces","nodes","persistentvolumeclaims","poddisruptionbudgets"]
I0522 06:42:43.559115       1 types.go:195] "Using all namespaces"
I0522 06:42:43.559140       1 server.go:256] "Metric allow-denylisting" allowDenyStatus="Excluding the following lists that were on denylist: "
W0522 06:42:43.559154       1 client_config.go:682] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.
I0522 06:42:43.559675       1 utils.go:70] "Tested communication with server"
I0522 06:42:43.562756       1 utils.go:75] "Run with Kubernetes cluster version" major="1" minor="36" gitVersion="v1.36.1+k3s1" gitTreeState="clean" gitCommit="a9663261a7ff40522542485a6b2f81916b6d72f9" platform="linux/amd64"
I0522 06:42:43.562765       1 utils.go:76] "Communication with server successful"
I0522 06:42:43.562792       1 server.go:288] "Using annotations allowlist" annotationsAllowList={}
I0522 06:42:43.562801       1 server.go:293] "Using labels allowlist" labelsAllowList={}
I0522 06:42:43.563252       1 server.go:390] "Started metrics server" metricsServerAddress="[::]:8080"
I0522 06:42:43.563290       1 server.go:379] "Started kube-state-metrics self metrics server" telemetryAddress="[::]:8081"
I0522 06:42:43.563306       1 metrics_handler.go:110] "Autosharding disabled"
I0522 06:42:43.563510       1 tls_config.go:354] "Listening on" address="[::]:8080"
I0522 06:42:43.563539       1 tls_config.go:357] "TLS is disabled." http2=false address="[::]:8080"
I0522 06:42:43.563575       1 tls_config.go:354] "Listening on" address="[::]:8081"
I0522 06:42:43.563614       1 tls_config.go:357] "TLS is disabled." http2=false address="[::]:8081"
I0522 06:42:43.565920       1 builder.go:294] "Active resources" activeStoreNames="certificatesigningrequests,configmaps,cronjobs,daemonsets,deployments,endpointslices,horizontalpodautoscalers,ingresses,jobs,leases,limitranges,mutatingwebhookconfigurations,namespaces,networkpolicies,nodes,persistentvolumeclaims,persistentvolumes,poddisruptionbudgets,pods,replicasets,replicationcontrollers,resourcequotas,secrets,services,statefulsets,storageclasses,validatingwebhookconfigurations,volumeattachments"
E0522 06:42:43.945655       1 cronjob.go:254] "Observed a panic" panic="failed to parse cron job schedule '*/60 * * * *': step (60) must be less than range size (60): \"*/60\"" panicGoValue="&fmt.wrapError{msg:\"failed to parse cron job schedule '*/60 * * * *': step (60) must be less than range size (60): \\\"*/60\\\"\", err:(*errors.errorString)(0x3211ff322b50)}" stacktrace=<
	goroutine 32 [running]:
	k8s.io/apimachinery/pkg/util/runtime.logPanic({0x2645e20, 0x3211ff880780}, {0x2125900, 0x3211ff686420})
		/go/pkg/mod/k8s.io/apimachinery@v0.35.4/pkg/util/runtime/runtime.go:132 +0xbc
	k8s.io/apimachinery/pkg/util/runtime.handleCrash({0x2646330, 0x3211fee0ea80}, {0x2125900, 0x3211ff686420}, {0x0, 0x0, 0x3211ff1835d0?})
		/go/pkg/mod/k8s.io/apimachinery@v0.35.4/pkg/util/runtime/runtime.go:107 +0x116
	k8s.io/apimachinery/pkg/util/runtime.HandleCrashWithContext({0x2646330, 0x3211fee0ea80}, {0x0, 0x0, 0x0})
		/go/pkg/mod/k8s.io/apimachinery@v0.35.4/pkg/util/runtime/runtime.go:78 +0x55
	panic({0x2125900?, 0x3211ff686420?})
		/usr/local/go/src/runtime/panic.go:860 +0x13a
	k8s.io/kube-state-metrics/v2/internal/store.cronJobMetricFamilies.func10(0x3211fef8e608)
		/go/src/k8s.io/kube-state-metrics/internal/store/cronjob.go:254 +0x18d
	k8s.io/kube-state-metrics/v2/internal/store.cronJobMetricFamilies.wrapCronJobFunc.func23({0x2491340?, 0x3211fef8e608})
		/go/src/k8s.io/kube-state-metrics/internal/store/cronjob.go:331 +0x4e
	k8s.io/kube-state-metrics/v2/pkg/metric_generator.(*FamilyGenerator).Generate(0x3211ff1837e8, {0x2491340?, 0x3211fef8e608?})
		/go/src/k8s.io/kube-state-metrics/pkg/metric_generator/generator.go:73 +0x2f
	k8s.io/kube-state-metrics/v2/internal/store.(*Builder).buildStores.ComposeMetricGenFuncs.func1({0x2491340, 0x3211fef8e608})
		/go/src/k8s.io/kube-state-metrics/pkg/metric_generator/generator.go:118 +0x12e
	k8s.io/kube-state-metrics/v2/pkg/metrics_store.(*MetricsStore).Add(0x3211ff50c5a0, {0x2491340, 0x3211fef8e608})
		/go/src/k8s.io/kube-state-metrics/pkg/metrics_store/metrics_store.go:64 +0x65
	k8s.io/kube-state-metrics/v2/pkg/metrics_store.(*MetricsStore).Replace(0x3211ff50c5a0, {0x3211ff322ae0, 0x1, 0x3211ff6863e0?}, {0x1?, 0x1?})
		/go/src/k8s.io/kube-state-metrics/pkg/metrics_store/metrics_store.go:121 +0x6b
	k8s.io/client-go/tools/cache.(*Reflector).watchList(0x3211fef1ad20, {0x2646330, 0x3211fee0ea80})
		/go/pkg/mod/k8s.io/client-go@v0.35.4/tools/cache/reflector.go:805 +0x9da
	k8s.io/client-go/tools/cache.(*Reflector).ListAndWatchWithContext(0x3211fef1ad20, {0x2646330, 0x3211fee0ea80})
		/go/pkg/mod/k8s.io/client-go@v0.35.4/tools/cache/reflector.go:423 +0x208
	k8s.io/client-go/tools/cache.(*Reflector).RunWithContext.func1()
		/go/pkg/mod/k8s.io/client-go@v0.35.4/tools/cache/reflector.go:369 +0x2e
	k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1({0x43aa3c?, 0x3d68500?})
		/go/pkg/mod/k8s.io/apimachinery@v0.35.4/pkg/util/wait/backoff.go:233 +0x13
	k8s.io/apimachinery/pkg/util/wait.BackoffUntilWithContext.func1({0x2646330?, 0x3211fee0ea80?}, 0x425000?)
		/go/pkg/mod/k8s.io/apimachinery@v0.35.4/pkg/util/wait/backoff.go:255 +0x51
	k8s.io/apimachinery/pkg/util/wait.BackoffUntilWithContext({0x2646330, 0x3211fee0ea80}, 0x3211ff183ef8, {0x262da20, 0x3211ff4188c0}, 0x1)
		/go/pkg/mod/k8s.io/apimachinery@v0.35.4/pkg/util/wait/backoff.go:256 +0xe5
	k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0x2650948?, {0x262da20?, 0x3211ff4188c0?}, 0x55?, 0x12?)
		/go/pkg/mod/k8s.io/apimachinery@v0.35.4/pkg/util/wait/backoff.go:233 +0x46
	k8s.io/client-go/tools/cache.(*Reflector).RunWithContext(0x3211fef1ad20, {0x2646330, 0x3211fee0ea80})
		/go/pkg/mod/k8s.io/client-go@v0.35.4/tools/cache/reflector.go:368 +0x235
	k8s.io/client-go/tools/cache.(*Reflector).Run(...)
		/go/pkg/mod/k8s.io/client-go@v0.35.4/tools/cache/reflector.go:359
	created by k8s.io/kube-state-metrics/v2/internal/store.(*Builder).startReflector in goroutine 143
		/go/src/k8s.io/kube-state-metrics/internal/store/builder.go:640 +0x509
 >
panic: failed to parse cron job schedule '*/60 * * * *': step (60) must be less than range size (60): "*/60" [recovered, repanicked]

goroutine 32 [running]:
k8s.io/apimachinery/pkg/util/runtime.handleCrash({0x2646330, 0x3211fee0ea80}, {0x2125900, 0x3211ff686420}, {0x0, 0x0, 0x3211ff1835d0?})
	/go/pkg/mod/k8s.io/apimachinery@v0.35.4/pkg/util/runtime/runtime.go:114 +0x1a9
k8s.io/apimachinery/pkg/util/runtime.HandleCrashWithContext({0x2646330, 0x3211fee0ea80}, {0x0, 0x0, 0x0})
	/go/pkg/mod/k8s.io/apimachinery@v0.35.4/pkg/util/runtime/runtime.go:78 +0x55
panic({0x2125900?, 0x3211ff686420?})
	/usr/local/go/src/runtime/panic.go:860 +0x13a
k8s.io/kube-state-metrics/v2/internal/store.cronJobMetricFamilies.func10(0x3211fef8e608)
	/go/src/k8s.io/kube-state-metrics/internal/store/cronjob.go:254 +0x18d
k8s.io/kube-state-metrics/v2/internal/store.cronJobMetricFamilies.wrapCronJobFunc.func23({0x2491340?, 0x3211fef8e608})
	/go/src/k8s.io/kube-state-metrics/internal/store/cronjob.go:331 +0x4e
k8s.io/kube-state-metrics/v2/pkg/metric_generator.(*FamilyGenerator).Generate(0x3211ff6b17e8, {0x2491340?, 0x3211fef8e608?})
	/go/src/k8s.io/kube-state-metrics/pkg/metric_generator/generator.go:73 +0x2f
k8s.io/kube-state-metrics/v2/internal/store.(*Builder).buildStores.ComposeMetricGenFuncs.func1({0x2491340, 0x3211fef8e608})
	/go/src/k8s.io/kube-state-metrics/pkg/metric_generator/generator.go:118 +0x12e
k8s.io/kube-state-metrics/v2/pkg/metrics_store.(*MetricsStore).Add(0x3211ff50c5a0, {0x2491340, 0x3211fef8e608})
	/go/src/k8s.io/kube-state-metrics/pkg/metrics_store/metrics_store.go:64 +0x65
k8s.io/kube-state-metrics/v2/pkg/metrics_store.(*MetricsStore).Replace(0x3211ff50c5a0, {0x3211ff322ae0, 0x1, 0x3211ff6863e0?}, {0x1?, 0x1?})
	/go/src/k8s.io/kube-state-metrics/pkg/metrics_store/metrics_store.go:121 +0x6b
k8s.io/client-go/tools/cache.(*Reflector).watchList(0x3211fef1ad20, {0x2646330, 0x3211fee0ea80})
	/go/pkg/mod/k8s.io/client-go@v0.35.4/tools/cache/reflector.go:805 +0x9da
k8s.io/client-go/tools/cache.(*Reflector).ListAndWatchWithContext(0x3211fef1ad20, {0x2646330, 0x3211fee0ea80})
	/go/pkg/mod/k8s.io/client-go@v0.35.4/tools/cache/reflector.go:423 +0x208
k8s.io/client-go/tools/cache.(*Reflector).RunWithContext.func1()
	/go/pkg/mod/k8s.io/client-go@v0.35.4/tools/cache/reflector.go:369 +0x2e
k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1({0x43aa3c?, 0x3d68500?})
	/go/pkg/mod/k8s.io/apimachinery@v0.35.4/pkg/util/wait/backoff.go:233 +0x13
k8s.io/apimachinery/pkg/util/wait.BackoffUntilWithContext.func1({0x2646330?, 0x3211fee0ea80?}, 0x425000?)
	/go/pkg/mod/k8s.io/apimachinery@v0.35.4/pkg/util/wait/backoff.go:255 +0x51
k8s.io/apimachinery/pkg/util/wait.BackoffUntilWithContext({0x2646330, 0x3211fee0ea80}, 0x3211ff6b1ef8, {0x262da20, 0x3211ff4188c0}, 0x1)
	/go/pkg/mod/k8s.io/apimachinery@v0.35.4/pkg/util/wait/backoff.go:256 +0xe5
k8s.io/apimachinery/pkg/util/wait.BackoffUntil(0x2650948?, {0x262da20?, 0x3211ff4188c0?}, 0x55?, 0x12?)
	/go/pkg/mod/k8s.io/apimachinery@v0.35.4/pkg/util/wait/backoff.go:233 +0x46
k8s.io/client-go/tools/cache.(*Reflector).RunWithContext(0x3211fef1ad20, {0x2646330, 0x3211fee0ea80})
	/go/pkg/mod/k8s.io/client-go@v0.35.4/tools/cache/reflector.go:368 +0x235
k8s.io/client-go/tools/cache.(*Reflector).Run(...)
	/go/pkg/mod/k8s.io/client-go@v0.35.4/tools/cache/reflector.go:359
created by k8s.io/kube-state-metrics/v2/internal/store.(*Builder).startReflector in goroutine 143
	/go/src/k8s.io/kube-state-metrics/internal/store/builder.go:640 +0x509
```

--- a/charts/kubewarden-controller/questions.yaml
+++ b/charts/kubewarden-controller/questions.yaml
@@ -14,7 +14,7 @@ questions:
     group: "Audit checks"
   - variable: "auditScanner.cronJob.schedule"
     type: string
-    default: "*/60 * * * *"
+    default: "0 * * * *"
     show_if: auditScanner.enable=true
     label: Schedule
     description: |

--- a/charts/kubewarden-controller/values.yaml
+++ b/charts/kubewarden-controller/values.yaml
@@ -336,7 +336,7 @@ auditScanner:
     tag: v1.36.0-beta.1
     pullPolicy: IfNotPresent
   cronJob:
-    schedule: "*/60 * * * *" # every 60 minutes
+    schedule: "0 * * * *" # every 60 minutes
     failedJobsHistoryLimit: 5
     successfulJobsHistoryLimit: 3
   containerRestartPolicy: Never

